src/commands/utils/detectors.ts

```diff
@@ -98,7 +98,7 @@ export async function getDetectorInfo(
             clusterNode.name,
             "detectors",
             detectorName,
-            "2019-08-01",
+            "2024-05-01",
```

Comment:
👍 nice, do you have handy link for archiving of all past versions of this API version, if so please share here.